### DISCUSSION TITLE
Auto Start

### DISCUSSION THREAD

**Discussion by cpt1971** [yuezk/GlobalProtect-openconnect#171] [Q&A] [answered]
Hi,

Is there away to stop the app from auto starting (start hidden in the system tray)?

By that I mean on login stop the app from displaying the UI so keeping it hidden in the system tray.

If you only need the Global Protect client on the odd occasion it's a little anowing.

Many thanks,
Chris

**Comment by yuezk** [comment, marked as answer]
Actually, I didn't add support for the client auto start. It might be the behavior of the OS. Which OS are you using?

**Reply by cpt1971** [reply]
Hi, Many thanks for getting back to me. I should firstly say a big thank you for writting such an excellent application.

So far I have only tried it on openSUSE 15.4 Leap (Plasma5) and I had to do the following to make it work:

zypper ar http://download.opensuse.org/repositories/network/15.4/ network

[I needed to remove this package to allow a vendor change] 
zypper rm openssh-askpass-gnome
 
zypper --non-interactive  dup --allow-vendor-change  --from network

[install openconnect]

zypper ar https://download.opensuse.org/repositories/home:/yuezk/15.4/ vpn

zypper in globalprotect-openconnect

My next test will be on Kubuntu 22.04 LTS as I tend to float between the two distros.

Many thanks once again,
Chris

**Reply by cpt1971** [reply]
Totaly my fault, the app was auto starting because I had not set my test vm to "Start with an empty session" which is why it was running the app at startup.

Apologies and many thanks,
Chris
